slatedb/src/oracle.rs

```diff
@@ -48,4 +62,14 @@ impl Oracle {
             ..self
         }
     }
+
+    pub(crate) fn track_last_committed_seq(&self, seq: u64) {
+        self.last_committed_seq.store(seq);
+        if let Ok(mut tracker) = self.seq_tracker.write() {
```

Comment:
Hmmmm.. I wonder about perf on this. Seems we call the lock for every write. @KarinaMilet's patch (#787) just got rid of a bunch of locking in `batch_write.rs`. 🤔

Reply:
I feel like the overhead of acquiring a lock is really small, the problem is contention -- I admit that during memtable flushing we lock this to copy all the data in the seq-tracker into the manifest. That might take a tiny amount of time...

Not sure if you consider this a blocker, I can try to make it a concurrent data structure but I'm not sure the added complexity is worth the cost.

Reply:
Nah, not blocking. I think it shouldn't hurt too much. The batch writer is single threaded and l0 flushing only happens infrequently.